client/lib/signup/step-actions/index.js

```diff
@@ -286,6 +308,10 @@ export function createSiteWithCart( callback, dependencies, stepData, reduxStore
 		return;
 	}
 
+	if ( newSiteParams.options.enable_fse ) {
+		recordTracksEvent( 'calypso_fse_enrolled', { flow: flowToCheck } );
```

Comment:
@creativecoder I have one more change request related to stats after discussing this with @kurt213.

Could we add a `site_enrolled: true` prop for enrolled users, and also record `calypso_fse_enrolled` with `site_enrolled: false` prop for non-enrolled users. It's just for the purposes of easier segmentation in the Tracks Funnels.

Reply:
I've added the `site_enrolled` property and the `calypso_fse_enrolled` event is now triggered for all new sites created in /start, specifying site_enrolled: true/false depending on FSE enrollment.

I'm hesitant to move the event trigger out of the site creation step. The signup flow is intentionally modular, and technically that design step could be triggered by any flow, at any point, for any existing site, including multiple times for the same site. I think keeping the event tied to site creation gives us better parity with the /new flow, and prevents some confusion in limiting the event to being triggered once per site.

Reply:
The most likely way the user won't follow the `site creation -> site design` pathway is by going through the checkout.

1. Create a site at `/start`
2. Choose a non-free plan
3. Go to checkout
4. The user bails out of the usual flow e.g. empties their checkout basket, follows a link shown during an upsell, etc.

In these situations, the user doesn't land in the `site-setup` flow.

So in this situation would we like `calypso_fse_enrolled` to be recorded or not? I think we should because the user will have the default theme (which is Zoologist, a universal theme) and will have access to the site editor if they're in the test cohort. So really they _are_ enrolled.
So I think we should keep `calypso_fse_enrolled` where it currently is in the flow.

Reply:
Makes sense, let's keep it as is then.